src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -1808,6 +1808,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
             Return False
         End Function
 
+        Public Function IsInInactiveRegion(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As Boolean Implements ISyntaxFacts.IsInInactiveRegion
+            If syntaxTree Is Nothing Then
+                Return False
+            End If
+
+            Return syntaxTree.IsInInactiveRegion(position, cancellationToken)
```

Comment:
Both C# and VB implementations of `IsInInactiveRegion` seem to directly call into `syntaxTree.IsInInactiveRegion` with a preceding null check for syntax tree. Do we really need to add `IsInInactiveRegion` method to `ISyntaxFacts`?

Reply:
yes.  the calls they make are into their respective extension methods in the lang specific layers.  the ISYntaxFacts form allows a lang neutral feature to call this and have it work.